services/QuillLMS/app/models/user.rb

```diff
@@ -86,8 +86,11 @@ class User < ApplicationRecord
   STUDENT = 'student'
   STAFF = 'staff'
   SALES_CONTACT = 'sales-contact'
-  ROLES      = [TEACHER, STUDENT, STAFF, SALES_CONTACT]
-  SAFE_ROLES = [STUDENT, TEACHER, SALES_CONTACT]
+  INDIVIDUAL_CONTRIBUTOR = 'individual-contributor'
+  ONBOARDING_ROLES   = [STUDENT, TEACHER, INDIVIDUAL_CONTRIBUTOR]
+  TEACHER_INFO_ROLES = [TEACHER, INDIVIDUAL_CONTRIBUTOR]
+  ROLES              = [TEACHER, STUDENT, STAFF, SALES_CONTACT]
+  SAFE_ROLES         = [STUDENT, TEACHER, SALES_CONTACT]
```

Comment:
Should `INDIVIDUAL_CONTRIBUTOR` be here in `SAFE_RULES`?  I'm pretty sure that "safe", in this case, means that we allow users to specify this role for themselves, something we don't do for `STAFF` because it lets you into the admin tools.

Reply:
@anathomical since this isn't really a "role" in that the user still has a "teacher" role for their user entry, I didn't think it should be in the `SAFE_ROLES` array. Maybe it makes more sense to have the constants used for this feature stored on the `TeacherInfo` class since that's where this role is actually being stored?